util/find_missing_timeline_translations.js

```diff
@@ -4,16 +4,19 @@ import Regexes from '../resources/regexes';
 import { Timeline } from '../ui/raidboss/timeline';
 import { commonReplacement, partialCommonReplacementKeys } from '../ui/raidboss/common_replacement';
 
-const triggersFile = process.argv[2];
-const locale = process.argv[3];
-const localeReg = 'regex' + locale[0].toUpperCase() + locale[1];
-findMissing(triggersFile, locale, localeReg);
+let kTriggersFile;
```

Comment:
Sorry for the bikeshedding, but I'm going to say maybe don't introduce more k-prefixed constants.  I originally intended `k` prefix things as constants, in a Typescript `readonly` sense, like for arrays of data that would never change.  I think global variables set with `let` are not good candidates for this, and probably we should remove them all because it's confusing to everyone but me.

Also, can you make these not global? This is a race condition here because `findMissing` is an async function and sets these values.  I could see:

findMissing1: call sets values to 1, waits
findMissing2: call sets values to 2, waits
findMissing1: import finishes, global values wrong now??

I think it would be clearer if they were explicit too, but that's my fault for writing it like this.

Reply:
Only reason I went with this approach was because if I pass these via the functions they will exceed 100 characters. I have no problem changing it, but I was just trying to (in my mind) best follow what was there.

Reply:
Could put them in some sort of shorter named object too, if that helps? There sure are a lot of parameters.

Reply:
I opted for doing less touching and just moving the localeReg closer to where it's being used, which puts us just under the 100 character limit.